website/blog/2021/08/11/interview-TuZhengsong.md

```diff
@@ -1,145 +1,145 @@
 
-在一个平淡的下午，我们和正松连线聊了聊。因为疫情关系，居住在南京的他已经多日未外出。在没有和他聊天之前，我没有想到，胡子拉碴的他，竟然走过了这么曲折的路。
+On an uneventful afternoon, we chatted with Zhengsong by phone. Because of the epidemic, he has not been out for many days in Nanjing. Before I talked to him, I didn't expect that the bearded man had traveled such a winding road.
 
-## 大学，我的大学
+## University, my university
 
-> 我大学的时候学的是通信工程，每天最开心的事就是踢球。现在有时候会想，如果能带着现在的记忆回到大学时光就好了。
+> When I was in college, I studied communication engineering, and the happiest thing I did every day was to play soccer. Now sometimes I think that it would be nice to go back to my college days with the memories I have now.
 
-正松在大学期间里面其实没怎么接触过编程，他当年其实有 C 语言的编程课，但是因为忙着踢球，挂科了。可是，他挂科之后的补考也没过，导致重修。最尴尬的是，他重修也没过，后来怎么办呢？毕业之前还有毕业清考。这可不能再挂了，于是正松就去图书馆去借了一本 C 语言的书来看。
+Zhengsong actually didn't have much contact with programming during his college years, he actually had a C programming class, but he failed it because he was too busy playing soccer. However, he failed a make-up exam after failing the class, resulting in a retake. The most embarrassing thing is that he failed the retake as well, so what did he do afterwards? There was a graduation clearing exam before graduation. This can not be failed again, so Zhengsong went to the library to borrow a C language book to read.
 
-“我当时心想，C 语言真的这么难吗？”正松说。
+"I was thinking, "Is C really that hard?" Zhengsong said.
 
-然后他就一直看一直看，看着看着就突然发现编程好像有那么点意思，他看懂了！最后他在毕业前顺利通过了考试。在毕业之后，找到了一份做程序员的工作。
+Then he kept reading and reading and reading, and suddenly he realized that programming seemed to have some meaning, and he understood it! He finally passed the exam before graduating. After graduation, he found a job as a programmer.
 
-“我本来都准备毕业回家找个厂上班了。”，他半开玩笑地说。
+"I was ready to graduate and go home and find a factory to work." , he said half-jokingly.
 
-彼时，他可能不会想到如今的他会成为数个知名开源项目的 developer，参与一些支撑着千万级并发流量的基础设施的开发。
+At that time, he probably wouldn't have thought that today he would be the developer of several well-known open source projects, and involved in the development of some infrastructure that supports millions of concurrent traffic.
 
-## 出象牙塔记
+## Out of the Ivory Tower
 
-> 在工业界，真实的业务场景和学校里学的很不一样的
+> In industry, the real business scenario is very different from what you learn in school.
 
-正松的第一份工作就职于一家做穿戴设备的创业公司，从事智能手环的研发工作。正松和他的前同事们需要通过处理从智能手环上的获取的各种传感器数据，来完成对用户的心率、运动量、血压等的监测。
+Zhengsong's first job was at a wearable device startup, working on the development of smart bracelets. Zhengsong and his former colleagues had to process various sensor data from the smart bracelet to monitor the user's heart rate, exercise, blood pressure and so on.
 
-“工作蛮好玩的，因为在工业界，真实的业务场景和学校里学的很不一样的，很多东西都得从头开始学”，正松说。
+"It was fun because in industry, the real business scenario is very different from what you learn in school, and a lot of things have to be learned from scratch," Masatsu said.
```

Comment:
```suggestion
"It was fun because in industry, the real business scenario is very different from what you learn in school, and a lot of things have to be learned from scratch," Zhengsong said.
```